Cargo.toml

```diff
@@ -26,7 +26,7 @@ bytemuck = { version = "1.4.1", default-features = false, features = [
 serde = { version = "1.0", default-features = false, features = [
     "derive",
 ], optional = true }
-num-traits = { version = "0.2.14", default-features = false, features = ["libm"], optional = true }
+num-traits = {git = "https://github.com/zommiommy/num-traits.git", default-features = false, features = ["libm"], optional = true }
```

Comment:
Don't change a cargo version dependency to a git dependency for any public crate please.